src/chrome/playSound.js

```diff
@@ -0,0 +1,20 @@
+import { SOUND_URL, SOUND_REPEAT } from '../constants';
+
+// Get sound options and play if enabled
+
+export const getAndPlaySoundNotificationOption = (store, mainAudioPlayer) => {
```

Comment:
Actually looking at this again, there's really no need to pass in the whole `store` (since this doesn't set up any listeners or anything like that). Can you just pass in `soundUrl` and `repeat` as arguments?

Also you can probably just call the function `playSound` for simplicity.